New issue template: Update Truth (#2332)
dtcenter/METplus: docs/Contributors_Guide/continuous_integration.rst

.. figure:: figure/ci-doc-artifacts.png

.. _cg-ci-update-truth-data:

Update Truth Data (update_truth.yml)
------------------------------------

The METplus use case test truth data includes output from use cases that is
used to compare with new use case test results to flag any differences.
Differences can occur due to changes to the METplus wrappers
source code/configuration files or changes to any of its dependent
METplus components such as MET, METplotpy, METcalcpy, and METdataio.
Differences can also occur when a new use case is added, as the new use case
creates output that does not yet exist in the truth dataset.

Once all differences are confirmed to be expected,
the reference branch, e.g. develop-ref, needs to be updated. This triggers a

> :ref:`cg-ci-testing-workflow` that runs all of the use cases, then creates

Reviewer: ```suggestion
:ref:`cg-ci-testing-workflow` that runs all of the use cases, creates
```

> Docker images that contain the new truth data and are pushed to DockerHub.

Reviewer: ```suggestion
Docker images with the new truth data, and pushes them to DockerHub.
```
